build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="test/ProjectsToTest.props" />
+  <Import Project="test/TestPackageProjects.props" />
+
+  <Target Name="PrepareTests"
+          DependsOnTargets="Prepare">
```

Comment:
Init vs Prepare is the current structure of our build system.  From the CLI C# build code:

```
// All major targets will depend on this in order to ensure variables are set up right if they are run independently
        public static BuildTargetResult Init(BuildTargetContext c)
```

The idea is: Init always needs to run, even when saying "build.cmd -Targets Compile", it sets up properties that are universally needed.  Prepare does extra stuff that doesn't need to run on every build (download shared fx, zip templates, restore root/src).

Although, now that we have incremental builds with MSBuild (we didn't with the CLI build framework), this becomes less important of a distinction. You should depend on exactly which targets you need, and then incrementality will take care of the rest.